rm-datasource/src/main/java/org/apache/seata/rm/datasource/xa/ConnectionProxyXA.java

```diff
@@ -389,4 +394,8 @@ private void reportStatusToTC(BranchStatus status) {
         }
     }
 
+    private boolean checkReadOnly() throws SQLException {
```

Comment:
It might be more appropriate to change the method name to isReadOnly. checkXXX indicates an action and isXXX indicates a state.

Reply:
Done